apps/docs/docs/tutorials/how-to-deep-link.md

```diff
@@ -3,6 +3,51 @@ sidebar_position: 6
 ---
 # Deep link to Portal Bridge
 
+> Note: this feature could work differently between the different bridged experiences
+> that currently portal bridge offers. The documentation presented here works on https://portalbridge.com
+
+## Pre-select Assets and Preferred route
+
+|Query Param  | Description |
+|---|---|---|
+|`sourceChain`                   |A source chain that will be pre-selected |
+|`targetChain`                   |A target chain that will be pre-selected |
+|`sourceAsset` or `asset`        |It will be the key of an asset that is injected in configs, you could check [here](https://github.com/XLabs/portal-bridge-ui/blob/main/apps/connect/src/env/token-bridge.mainnet.ts#L505)|
```

Comment:
just double checking that its 'asset' and not 'Asset'

Reply:
yes, it is asset, and it is case sensitive